zipline/assets/asset_writer.py

```diff
@@ -81,7 +81,7 @@
     'root_symbol': None,
     'asset_name': None,
     'start_date': 0,
-    'end_date': 2 ** 62 - 1,
+    'end_date': np.iinfo(np.int64).max,
```

Comment:
It essentially reverting ee9c79a, to make this more clear. There was apparently some compatibility issue at some point, but I wasn't able to reproduce it.